src/classes/RLLP_OppRollup_TEST.cls

```diff
@@ -36,12 +36,6 @@
 */
 @isTest
 public with sharing class RLLP_OppRollup_TEST {
-
-    /*********************************************************************************************************
```

Comment:
Hi @ceiroa Can you please give me some context about why we need to delete this line?